lib/rules/no-test-support-import.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileOverview no importing of test files and no exporting in test files
+ */
+
+'use strict';
+
+const ERROR_MESSAGE_NO_IMPORT =
+  'Do not import a file from test-support into production code, only into files located in test.';
```

Comment:
`only into files located in test` -> `only into test files`?

Reply:
updated with this new wording.